dds/idl/be_global.cpp

```diff
@@ -348,6 +348,9 @@ BE_GlobalData::parse_args(long& i, char** av)
   static const char EXPORT_FLAG[] = "--export=";
   static const size_t EXPORT_FLAG_SIZE = sizeof(EXPORT_FLAG) - 1;
 
+  static const char TYPESEQUENCESUFFIC_FLAG[] = "--typeSequenceSuffix=";
```

Comment:
```suggestion
  static const char TYPESEQUENCESUFFIC_FLAG[] = "--type-sequence-suffix=";
```

To match the other long options.

Reply:
Changed